### PR TITLE
Sample worksets

### PR DESCRIPTION
**Pull request by cwulfman**
Addresses #4

### CODE REVIEW

File: doc/sample_worksets.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Native-authored Workset
+id: 6408a5e930000067055ce2e8
+volumes: 919
+description: >
+             A list of works authored
+             by Native Americans in HathiTrust
+author: SCWAReD
```
Comment (borice) [inline doc/sample_worksets.yaml:7]:
```suggestion
- name: Native-authored Workset
  id: 6408a5e930000067055ce2e8
  volumes: 919
  description:
             A list of works authored
             by Native Americans in HathiTrust
  author: SCWAReD
```

**Review by borice — CHANGES_REQUESTED** [review]
You should make the first one part of the list too...

File: doc/sample_worksets.yaml (hunk 1 above)
Comment (borice) [reply doc/sample_worksets.yaml:7]: Please write the code in such a way that this file could be fetched from anywhere ... locally, or remotely.
(i.e. use a url to reference the file, like `file:///blah.yaml` which could be `http://foo/blah.yaml`)

Comment (cwulfman) [reply doc/sample_worksets.yaml]: Good idea

File: doc/sample_worksets.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+- name: Native-authored Workset
+  id: 6408a5e930000067055ce2e8
+  volumes: 919
+  description: >
```
Comment (borice) [inline doc/sample_worksets.yaml:4]:
```suggestion
  description: 
```

**Review by borice — CHANGES_REQUESTED** [review]
No need for `>`

**Review by borice — APPROVED** [review]

**Pull request merged**